xdfile/puz2xd.py

```diff
@@ -43,7 +43,7 @@ def is_block(puz, x, y):
 
 
 def parse_puz(contents, filename):
-    rebus_shorthands = list("⚷⚳♇♆⛢♄♃♂♁♀☿♹♸♷♶♵♴♳⅘⅗⅖⅕♚♛♜♝♞♟⚅⚄⚃⚂⚁⚀♣♦♥♠+&%$@?*zyxwvutsrqponmlkjihgfedcba0987654321")
+    rebus_shorthands = list("zyxwvutsrqponmlkjihgfedcba⚷⚳♇♆⛢♄♃♂♁♀☿♹♸♷♶♵♴♳⅘⅗⅖⅕♚♛♜♝♞♟⚅⚄⚃⚂⚁⚀♣♦♥♠+&%$@?*0987654321")
```

Comment:
Ah, I think I see, this moves the lowercase letters to be last-allocated for a rebus, because in the case of a rebus+shading, we need the lowercase letters to indicate the shaded squares.  But then, how do we handle a shaded square with non A-Z within it?  Or has this just not come up yet?  (I suspect we'll need another mechanism for these, but that would require a change to the format which I'm not looking forward to).

Reply:
That's right regarding why I moved a-z. For the the case of a cell that is both rebus and special, you actually have a solution for this in the Readme: use a character in a-z as the rebus symbol. So e.g. a circled cell with fill "EXAMPLE" could have a rebus substitution "q" in the grid, and in the headers you'd have "Rebus: q=EXAMPLE". This will generally solve the problem, but in the case of a pangram puzzle that uses every A-Z, there would be no a-z available as rebus symbols as it would not be possible to tell whether a given special cell has as its fill the literal a-z character or that character as a rebus symbol. That concern is why I held off implementing that logic in this PR (also, laziness). I could patch it in if desired. Or yes, otherwise to solve this generally I think requires a nontrivial change to xd which would require more thought.

Also, a reminder that the puz convention is to code any kind of special cell as a circle. Therefore this PR will still wrongly parse shaded cells (as they might appear in print or web versions) as circles, but only because this is how they're represented in the puz.

Reply:
Thanks for reminding me :)  I think in the case you mentioned we can still use a-z for shaded; we might just have to do a=A or otherwise be clever.  As I don't think this matters for any existing puzzles, I'll take this as-is and we can adjust later when need be.